lua/nvim-treesitter/parsers.lua

```diff
@@ -457,6 +457,7 @@ list.glimmer = {
   readme_name = "Glimmer and Ember",
   maintainers = { "@alexlafroscia" },
   filetype = "handlebars",
+  used_by = { "html.handlebars" },
```

Comment:
This won't work, as with https://github.com/nvim-treesitter/nvim-treesitter/pull/1771/ it always extracts the first part (`html`). The code could be adapted to look the full name in the filetype table first or just drop that "feature" of returning the first part and always require explicit filetypes https://github.com/nvim-treesitter/nvim-treesitter/pull/1771#discussion_r700309654

Reply:
Ahh!! That makes sense! Thank you @stsewd, I'll take a look at that comment and see if I can give it a go...

Reply:
@stsewd - I've updated to your suggestion, mind taking another look?